### DISCUSSION TITLE
dotnet-fm: How to pass in multiple -t tags?

### DISCUSSION THREAD

**Discussion by vmandic** [fluentmigrator/fluentmigrator#1676] [Q&A] [answered]
**Describe the bug**
Running dotnet-fm does not take into account multiple -t tags, all are ignored. Only untagged ones are used.

**To Reproduce**

```bash
dotnet build -v:m -c Release ./App.SqlMigrations -o ./publish \
&& echo "Running SQL migrations..." \
&& DOTNET_ROLL_FORWARD=LatestMajor dotnet fm migrate -p mysql5 \
-a "./publish/App.SqlMigrations.dll" \
-c "$connString" \
-t "Tag1" \
-t "Tag2"
```

**Expected behavior**
Bot untagged, and Tag1 and Tag2 C# migrations should be included.

**Information (please complete the following information):**
 - OS: macos 12.5
 - Platform NET Core SDK 7.0.203
 - FluentMigrator version 3.3.2
 - FluentMigrator runner dotnet-fm
 - Database Management System mysql
 - Database Management System Version mariadb:10.3.37

**Additional context**
None.

**Comment by jzabroski** [comment, marked as answer]
-t | --tag is an IEnumerable<string> and takes a space-separated list of arguments after the initial parameter name, i.e., `-t Tag1 Tag2`

https://github.com/fluentmigrator/fluentmigrator/blob/5fa7ce26d943159f90ae164660a8d5d8f5edf17f/src/FluentMigrator.DotNet.Cli/Commands/MigrationCommand.cs#L44

We use Nate McMaster's Command Line library, for which you can read here: https://natemcmaster.github.io/CommandLineUtils/docs/arguments.html#variable-numbers-of-arguments

Note, it is unfortunate the longform name of the parameter is just --tag, it should really be --tags.  Happy to accept a PR to the fluentmigrator/documentation repo to provide examples.

**Reply by vmandic** [reply]
Thanks @jzabroski! I am happy to see you back and well!
